tms_objects2.erb

```diff
@@ -842,6 +858,11 @@ objects_main_query.each { |r|
   ownerAltName.each { |r1|
     OwnerAltName << r1
   }
+  objTEauthor3 = tmsclient.execute("SELECT * from [Coboat_ObjTEauthor3] where ID ="+ObjectID.to_s).each
```

Comment:
add a comment:
#5011 - test of multiple text entries, including non gallery labels with event actors